packages/sdk/src/index.ts

```diff
@@ -15,3 +15,5 @@ export { utf8ToBytes, bytesToUtf8 } from "@waku/utils/bytes";
 export * from "./create.js";
 export * as waku from "@waku/core";
 export * as utils from "@waku/utils";
+export * from "@waku/interfaces";
```

Comment:
Re-exporting on the root level to provide convenient import on the consumer side, e.g:
`import type { RelayNode } from "@waku/sdk";`